CHANGELOG.md

```diff
@@ -12,7 +12,34 @@ accidentally triggering the load of a previous DB version.**
 * #4397 Support intervals with month component when constifying now()
 
 **Thanks**
-@nikugogoi for reporting a bug with CTEs and upserts on distributed hypertables
+* @nikugogoi for reporting a bug with CTEs and upserts on distributed hypertables
+
+
+## 2.7.1 (2022-07-01)
+
+This release is patch release. We recommend that you upgrade at the
+next available opportunity.
```

Comment:
I think historically we say "high, medium, low" priority?   I'm not sure what "upgrade at the next available opportunity means"?  Maybe there is a bit more context/explanation we could give?

Reply:
We do not seem to do this for patch releases. At least 2.6.1 does not have this priority added, which is where I copied this from.

Reply:
Also "This release is patch release." sounds a bit odd to me.

Reply:
Open for suggestions on how to word this. :)

Reply:
Changed the wording to "This release is a patch release".